docs/en/ingest-management/fleet/fleet-settings-output-elasticsearch.asciidoc

```diff
@@ -81,6 +81,16 @@ output is set in the <<agent-policy,agent policy>>.
 | When this setting is on, {agent}s use this output to send <<monitor-elastic-agent,agent monitoring data>> if no other output is set in the <<agent-policy,agent policy>>.
 
 Sending monitoring data to a remote {es} cluster is currently not supported.
+
+// =============================================================================
+
+|
+[id="es-agent-performance-tuning"]
+**Performance tuning**
+
+| Choose one of the menu options to tune your {agent} performace when sending data to an {es} output. You can optimize for throughput, scale, latency, or you can choose a balanced (the default) set of performance specifications. Refer to <<es-output-settings-performance-tuning-settings,Performance tuning settings>> for details about the setting values and their potential impact on performance.
```

Comment:
Typo fix

```suggestion
| Choose one of the menu options to tune your {agent} performance when sending data to an {es} output. You can optimize for throughput, scale, latency, or you can choose a balanced (the default) set of performance specifications. Refer to <<es-output-settings-performance-tuning-settings,Performance tuning settings>> for details about the setting values and their potential impact on performance.
```

Reply:
Thanks for catching that! Fixed via [1df7941](https://github.com/elastic/ingest-docs/commit/1df7941f6e9ead235001dd4eb85d90e33389a2fe)